source/includes/administration/_applied_princings.md

```diff
@@ -0,0 +1,223 @@
+### Applied Pricings
+
+The applied pricing allows the assignment of pricing to organization with a determined scope.
+
+<!-------------------- LIST APPLIED PRICINGS -------------------->
+#### List applied pricings
+
+`GET /applied_pricings`
+
+Retrieves a list of applied pricings configured in the system.
+
+```shell
+# Retrieve applied pricing list
+curl "https://cloudmc_endpoint/rest/applied_pricings" \
+   -H "MC-Api-Key: your_api_key"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "pricingDefinition": {
+        "supportedCurrencies": [
+          "CAD",
+          "USD"
+        ],
+        "organization": {
+          "id": "23910576-d29f-4c14-b663-31d728ff49a5"
+        },
+        "name": {
+          "en": "Simple pricing",
+          "fr": "Facturaction simple"
+        },
+        "changes": [],
+        "description": {},
+        "id": "f786b9c5-177b-4cbc-9011-331282485d05",
+        "effectiveDate": "2020-06-04T17:55:17Z"
+      },
+      "endDate": "2021-06-05T23:59:59.999Z",
+      "organization": {
+        "name": "System",
+        "id": "23910576-d29f-4c14-b663-31d728ff49a5"
+      },
+      "billingDay": 5,
```

Comment:
From my end (in list-applied-pricings branch) I think I don't have "billingDay" attribute. Should there be a billing day attribute?

Reply:
I added in my PR. It will not be part of the UI screen, but will be part of the attributes for the API call